.gitignore

```diff
@@ -43,7 +192,7 @@ htmlcov/
 .cache
 nosetests.xml
 coverage.xml
-*.cover
+*,cover
```

Comment:
@Gagaro , extension is not a problem here. I see that . has been replaced with a , in this commit. I didn't want it to ignore all files because of the comma over there. Other than that, the pull request looks good. I will go ahead and accept it.